packages/react/error-boundary/src/ErrorBoundary.tsx

```diff
@@ -67,31 +66,17 @@ class BaseErrorBoundary extends Component<ErrorBoundaryProps, ErrorBoundaryState
     onError?.(error, info);
   }
 
-  resetState() {
-    this.updatedWithError = false;
-    this.setState(initialState);
-  }
-
```

Comment:
I remove this method. if no updatedWithError field, resetState is not anymore meaningful too.
I thought `this.setState(initialState)` is better to express what this is.